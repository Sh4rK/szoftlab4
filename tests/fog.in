loadMap test.map
loadMission test.mission
buildTower 7 7
setFog 1
step 1
listEnemies
exit