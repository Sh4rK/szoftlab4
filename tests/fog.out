(7.0;1.0) -
1 75.0 (7.5;7.5)
1 75.0 (7.8;7.8)
1 75.0 (8.0;8.0)
1 75.0 (8.3;8.3)
1 75.0 (8.6;8.6)
1 75.0 (8.8;8.8)
